frontend/public/components/factory/list.tsx

```diff
@@ -194,6 +195,32 @@ const listFilters = {
   },
 };
 
+/* Append filters from plugins
+
+TODO(mlibra):
+Some objects (like VirtualMachines) require to query additional objects
+to fully asses state (like pods and CRs in the case of a VirtualMachine).
+Recent filtering logic can handle just a single "listed" object.
+We will need to find a way how to pass additional resources there.
+*/
+(function () {
+  const baseListCount = Object.keys(listFilters).length;
+
+  plugins.registry.getResourceListPages()
+    .filter(rlp => !!rlp.properties.filter)
+    .forEach(rlp => {
+      if (listFilters[rlp.properties.model.kind]) {
+        // eslint-disable-next-line no-console
+        console.warn(`List filter for ${rlp.properties.model.kind} already defined, skipping.`);
```

Comment:
Move this check to a unit test. 
fyi @vojtechszocs

Reply:
Yeah, this is exactly the purpose of #1708 - run extension checks as part of `yarn test` while keeping the runtime browser console log cleaner.